source/Cosmos.System2/Graphics/Image.cs

```diff
@@ -43,6 +44,57 @@ protected Image(uint width, uint height, ColorDepth color)
             Height = height;
             Depth = color;
         }
+
+        /// <summary>
+        /// Resizes the bitmap
+        /// </summary>
+        /// <param name="NewW">the new width of the bitmap</param>
+        /// <param name="NewH">the new height of the bitmap</param>
+        public void Resize(uint NewW, uint NewH)
+        {
+
```

Comment:
Please do not include empty lines at the beginning and end of functions. Also keep the naming scheme of parameters the same as the rest of the file